### ISSUE TITLE
coerceTypes overrides property value from previous property

### ISSUE THREAD

**Issue by atfzl** [ajv-validator/ajv#152]
``` javascript
var Ajv = require('ajv');
var ajv = Ajv({ coerceTypes: true });

var schema = {
  type: 'object',
  properties: {
    foo: {
      type: 'number'
    },
    bar: {
      type: 'number'
    }
  }
};

var ob = {
  foo: '123',
  bar: 'asdad'
};

var valid = ajv.validate(schema, ob);

console.log(ob); // { foo: 123, bar: 123 }

console.log(ajv.errorsText()); // No errors
```

**Comment by epoberezkin** [comment]
fixed in 3.8.7. Thanks!

**Comment by atfzl** [comment]
@epoberezkin Thank you
